go.mod

```diff
@@ -68,7 +68,10 @@ require (
 	github.com/googleapis/enterprise-certificate-proxy v0.3.4 // indirect
 	github.com/googleapis/gax-go/v2 v2.14.0 // indirect
 	github.com/grpc-ecosystem/grpc-gateway/v2 v2.23.0 // indirect
+	github.com/hashicorp/errwrap v1.1.0 // indirect
+	github.com/hashicorp/go-multierror v1.1.1 // indirect
 	github.com/hashicorp/golang-lru/v2 v2.0.7 // indirect
+	github.com/jlaffaye/ftp v0.2.0 // indirect
```

Comment:
this should not be part of the gofr go.mod

Reply:
As the go.mod are removed in examples, isn't normal things are going up, and are added to main one no?

If this is the problem, we can leave a go.mod at the root of examples

Another solution is to move the examples folder into a testdata folder, as it's a special folder

Reply:
For examples like http-server we are using the datasources(sql,redis) which are already part part of `go.mod`.
and in example like add-filestore, we had a different go.mod as we were importing ftp which is not part of the gofr `go.mod` but it is implemented as an external datasource.

Reply:
Yes, exactly. So I agree there is something strange

Reply:
Will do one thing, we will remove the go.mod from **http-server example** only so that this unwanted dependency doesn't get added to GoFr's `go.mod`